docs/configuration_options.md

```diff
@@ -3775,3 +3775,11 @@ Vector Store indexing threads (if not set, defaults to number of CPU cores on VS
 **default:** N/A
 
 **type:** int
+
+## **scaling** / SCT_CLUSTER_SCALING
```

Comment:
this is auto generated docs, those kind of things should be added to `sct_config.py`, and that it should be generated based on that